test/proper_tests.erl

```diff
@@ -1107,16 +1107,15 @@ setup_test_() ->
 -endif.
 
 adts_test_() ->
-    [{timeout, 42,	% for Kostis' old laptop
-      ?_passes(?FORALL({X,S},{integer(),?SET()},
+    [{timeout, 20,	% for Kostis' old laptop
+      ?_passes(?FORALL({X,S}, {integer(),?SET()},
 		       sets:is_element(X,sets:add_element(X,S))), [20])},
-     {timeout, 84,	% for 18.x (and onwards?) on slow machines
-      ?_passes(?FORALL({X,Y,D},
-		       {integer(),float(),?DICT(integer(),float())},
+     {timeout, 40,	% for 18.x (and onwards?)
+      ?_passes(?FORALL({X,Y,D}, {integer(),float(),?DICT(integer(),float())},
 		       dict:fetch(X,dict:store(X,Y,eval(D))) =:= Y), [30])},
-     ?_fails(?FORALL({X,D},
-	     {boolean(),?DICT(boolean(),integer())},
-	     dict:erase(X, dict:store(X,42,D)) =:= D))].
+     {timeout, 20,      % seems to be needed on slow machines (ARM)
```

Comment:
I found that this does pass on our aarch64 builder, but still fails on our armv7hl builder. If I change this 20 to 256 it passes on all Fedora architectures. I also tried 128 and that failed on armv7hl as well. So somewhere between 128 and 256 is probably the sweet spot, but if you think 256 is an ok value here I'm happy to just go with that.